packages/playwright-ct-react/registerSource.mjs

```diff
@@ -76,23 +76,27 @@ async function __pwResolveComponent(component) {
 }
 
 /**
- * @param {JsxComponent | JsxComponentChild} component
+ * @param {JsxComponentChild} child
  */
-function __pwRender(component) {
-  if (!isComponent(component))
```

Comment:
I think removing this check breaks the following code: `await mount(23)` or `await component.update('some text')`. Not sure whether we actually supported this use case, wdyt?

Reply:
This case is not supported because `23` doesn't conform to the expected `JSX.Element` type. Only JSX elements can be passed with the `mount` and `update` functions

Reply:
Thank you for clarification!